Assets/MixedRealityToolkit.Tests/TestUtilities.cs

```diff
@@ -163,5 +165,11 @@ public static void ShutdownMixedRealityToolkit()
             return ScriptableObject.CreateInstance<T>();
 #endif
         }
+
+        public static void AssertAboutEqual(Vector3 actual, Vector3 expected, string message)
```

Comment:
Unity has something for this, have a look at Vector3EqualityComparer here: https://docs.unity3d.com/2017.4/Documentation/Manual/PlaymodeTestFramework.html.

Reply:
thanks! I should use that for manipulationhandler as well - doing the manual magnitude compare there as well